caddy/caddymain/run.go

```diff
@@ -40,6 +40,8 @@ func init() {
 	flag.StringVar(&revoke, "revoke", "", "Hostname for which to revoke the certificate")
 	flag.StringVar(&serverType, "type", "http", "Type of server to run")
 	flag.BoolVar(&version, "version", false, "Show version")
+	flag.StringVar(&caddytls.HTTPChallengePort, "external-port-80-is-local-port", "80", "Local port exposed externally as port 80")
+	flag.StringVar(&caddytls.TLSSNIChallengePort, "external-port-443-is-local-port", "443", "Local port exposed externally as port 443")
```

Comment:
Can you clarify what you mean by "I don't think these should change"? Do you mean that Caddy should never try to open local ports other than 80 or 443 to respond to ACME challenges? I'm specifically interested in responding to ACME challenges on alternative ports because we have a NAT router forwarding i.e. external_ip:80 to internal_ip:8080. Or do you mean that these values should be stored in a different set of variables?